profiles.md

```diff
@@ -0,0 +1,208 @@
+  "uuid": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "name": "msmith@samltest.id",
+  "handle": null,
+  "metadata": {
+    "eperson.firstname": [{
+        "value": "Mortimer",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "eperson.lastname": [{
+        "value": "Smith",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "netid": "msmith@samltest.id",
+  "lastActive": "2019-10-25T08:05:46.530+0000",
+  "canLogIn": true,
+  "email": "msmith@samltest.id",
+  "requireCertificate": false,
+  "selfRegistered": false,
+  "type": "eperson"
+}
+```
+
+It would respond with:
+* 200 OK - Returning the eperson if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the eperson
+* 404 Not found - if the eperson doesn't exist
+
+## Creating a new Profile
+**POST /api/cris/profiles?eperson=<:eperson-uuid>**
+
+Create a new profile for the specified person or for the current user. Only an administrator can create a profile for another eperson.
+
+Parameters:
+* eperson-uuid (optional): the uuid of eperson for which the profile is to be created; if not specified, the eperson is the authenticated one
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if for the specified eperson a profile already exists
+* 422 Unprocessable Entity - if the specified person does not exist
+
+In case of response with status 201 the endpoint returns the newly created resource, while in case of 409 it returns the resource that generated the conflict.
+
+To create a new profile use 
+```bash
+curl -i -X POST ${dspace-url}/api/cris/profiles?eperson=eb645ef8-1373-41eb-bf67-6afcea7e2069 -H "Content-Type:application/json"
+```
+
+## Hide/unhide a profile
+**PATCH /api/cris/profiles/<:eperson-uuid>**
+
+This operation allow to change the visibility of one profile. Only an administrator can modify the profile of another eperson.
+
+To hide or unhide a profile use 
+```bash
+curl -i -X PATCH ${dspace-url}/api/cris/profiles/eb645ef8-1373-41eb-bf67-6afcea7e2069 --data '[ { "op": "replace", "path": "/visible", "value": true }]' -H "Content-Type:application/json"
+```
+
+Status codes:
+
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 404 Not found - if a profile for the specified eperson does not exist
+
+If the operation succeed the endpoint returns the updated resource.
+
+## Delete a profile
+**DELETE /api/cris/profiles/<:eperson-uuid>**
+
+Delete the profile related to the given eperson. Only an administrator can delete the profile of another eperson.
```

Comment:
Same here, may want to reword to say it can only be done by the EPerson themselves or an Administrator.

I also have a question. Does `DELETE` in this case remove the Person Entity from the system entirely?  Or is it just removing the linkage between the Person Entity and the EPerson?  There may be scenarios where you just want to remove the link.  For example, if the user leaves your organization, you may want to *delete* their EPerson (and profile link), but *keep* their Person Entity (as that Person Entity may be linked to several publications, etc).